src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const NewRobot = {};
```

Comment:
variables should be in camelCase
```suggestion
  const newRobot = {};
```